plugins/package-managers/node/src/main/kotlin/npm/Npm.kt

```diff
@@ -126,6 +132,8 @@ class Npm(override val descriptor: PluginDescriptor = NpmFactory.descriptor, pri
         val project = parseProject(definitionFile, analysisRoot)
         val projectModuleInfo = listModules(workingDir, issues).undoDeduplication()
 
+        requestAllPackageDetails(projectModuleInfo) // Warm-up the cache to speed-up processing.
```

Comment:
Nit: Maybe put the comment above the line, as it stands on its own already anyway?

Reply:
Hmm, I would have kept the dot at the end of the sentence, though, instead of changing it to a colon. IMO it's clear that the comment refers to the next line, as comment and code are grouped anyway.

Reply:
mh, ok.